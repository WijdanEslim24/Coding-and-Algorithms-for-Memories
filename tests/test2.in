6
bxydbim
txwdbcm
ndbcimd
abmximd
aynddci
xwndbcd
8