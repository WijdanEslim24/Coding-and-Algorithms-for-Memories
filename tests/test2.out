SCS: abmtxywndbdcimd
SCS Length: 15